otx/algorithms/common/adapters/mmcv/models/backbones/torchvision_backbones.py

```diff
@@ -0,0 +1,247 @@
+"""Torchvision Model Backbone Class Generation.
```

Comment:
I'm not sure whether the location of `torchvision_backbones.py`  is right or not. 
Because it is a kind of `torch` series, I think `common/adapter/pytorch/` is more suitable for me.

Reply:
This is a code that converts the backbone class provided by `torchvision` to the `mmcv` backbone class and registers it in the `mmcv` registry. I think it's appropriate here because it's the code that makes it usable in `mmcv`.